samples/components/oauth/redshift.azure.iam-idc.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="utf-8"?>
+<pluginOAuthConfig>
+  <dbclass>redshift</dbclass>
+  <!-- For configs embedded in the connector package, don't prefix with "custom_". For external configs, always prefix with "custom_". -->
+  <oauthConfigId>custom_redshift_azure_iam_idc</oauthConfigId>
+  <clientIdDesktop>${clientID}</clientIdDesktop>
+  <clientSecretDesktop>${clientSecret}</clientSecretDesktop>
+  <redirectUrisDesktop>http://localhost:55556/Callback</redirectUrisDesktop>
+  <redirectUrisDesktop>http://localhost:55557/Callback</redirectUrisDesktop>
+  <redirectUrisDesktop>http://localhost:55558/Callback</redirectUrisDesktop>
+  <redirectUrisDesktop>http://localhost:55559/Callback</redirectUrisDesktop>
+  <!-- For multitenant apps use the common endpoint, for single tenant apps use the directory specific endpoint. -->
+  <authUri>https://login.microsoftonline.com/${tenant}/oauth2/v2.0/authorize</authUri>
+  <tokenUri>https://login.microsoftonline.com/${tenant}/oauth2/v2.0/token</tokenUri>
+  <scopes>openid</scopes>
+  <scopes>offline_access</scopes>
+  <scopes>email</scopes>
+  <!-- An example with a custom API, which was required at the time of writing for integration with AWS IAM IDC. -->
+  <scopes>api://${customAPI}/Redshift</scopes>
+  <capabilities>
+    <entry>
+      <key>OAUTH_CAP_REQUIRES_PROMPT_SELECT_ACCOUNT</key>
+      <value>true</value>
+    </entry>
+    <entry>
+      <key>OAUTH_CAP_REQUIRE_PKCE</key>
+      <value>true</value>
+    </entry>
+    <entry>
+      <key>OAUTH_CAP_PKCE_REQUIRES_CODE_CHALLENGE_METHOD</key>
+      <value>true</value>
+    </entry>
+    <entry>
+      <key>OAUTH_CAP_SUPPORTS_STATE</key>
+      <value>true</value>
+    </entry>
+    <entry>
+      <key>OAUTH_CAP_CLIENT_SECRET_IN_URL_QUERY_PARAM</key>
+      <value>true</value>
```

Comment:
okay last q from me, should this be consistent with the other azure config for redshift? false?

Reply:
No I don't think so. If there are multiple valid configs it's reasonable to show alternatives.

Reply:
I reverted the change to the existing example.